plugin.video.mazacka/main.py

```diff
@@ -88,11 +88,20 @@ def getRequest(url, udata=None, headers=httpHeaders):
 def checkStream(html):
     if html == '':
         xbmcgui.Dialog().ok(PLUGIN_ID, _lang(30003))
+        EXIT()
         return(False)
-    if html.find('Vys&#237;l&#225;n&#237; bude brzy zah&#225;jeno') != -1:
+    if html.find('is-inactive') != -1:
+        placeholder = re.search(r'(?:http\:|https\:)?\/\/[^><"]*\.jpg', html)
+        if placeholder: xbmc.executebuiltin('ShowPicture("{0}")'.format(placeholder.group()))
+        
         xbmcgui.Dialog().ok(PLUGIN_ID, _lang(30002))
+        EXIT()
         return(False)
     return(True)
+    
+def EXIT():
+     xbmc.executebuiltin('Dialog.Close(All)')
+     xbmc.executebuiltin("XBMC.ActivateWindow(Home)")
```

Comment:
XBMC. prefixes no longer work in matrix, since this is supported for a long time please drop it and use only `xbmc.executebuiltin("ActivateWindow(xxx)")`